CMakeLists.txt

```diff
@@ -78,7 +78,12 @@ find_package(BISON 2.6.1 REQUIRED)
 find_package(FLEX 2.6.1 REQUIRED)
 
 # This MUST be executed after BuildStatic since it sets Boost Static flags
-find_package(Boost REQUIRED COMPONENTS filesystem system date_time)
+set(_Boost_DEBUG on) 
```

Comment:
Why is this variable set?

Reply:
I was trying to fix the windows issue.

Reply:
From the CI the message is: `Could NOT find Boost (missing: program_options) (found version "1.75.0")`. Probably the fix is only to add the `boost-program-options` at the [build script](https://github.com/esbmc/esbmc/blob/master/.github/workflows/build.yml#L119) of the CI. 

If this solves the issue, remember to add [here](https://github.com/esbmc/esbmc/blob/master/.github/workflows/release.yml#L92) as well